CONTRIBUTING.md

```diff
@@ -246,6 +246,36 @@ This key identifies the section number from the portion of the ES6 standard that
 
 Read the [Test262 Technical Rationale Report](https://github.com/tc39/test262/wiki/Test262-Technical-Rationale-Report,-October-2017#specification-reference-ids) for reasoning behind deprecation.
 
+## Staging
+
+There is a `test/staging/` folder, containing tests that are subject to
+fewer requirements, in order to get tests running across implementations
+as early as possible, and promote interoperability of in-progress
+features.
+
+Tests in staging are not required to be split up into one test per file,
+or to conform to any particular style as long as they are runnable.
+In particular, mechanically-converted tests from implementations'
+private test suites are encouraged, as sharing them with other
+implementations promotes interoperability.
+
+These are the requirements for adding a test to staging:
+
+1. It tests a late-stage TC39 proposal or an existing language feature.
+2. It is correct.
+3. It is runnable using the usual test262 runner.
+4. It is reviewed by someone with permission to land tests in the
+   staging folder, not necessarily a test262 maintainer.
+5. If it requires the implementation to support ECMA-402, it must be in
+   `test/staging/intl402/` or a subfolder thereof.
+
+Tests are intended to live temporarily in staging.
+Once the feature they test has stabilized, contributors are encouraged
+to refine them to meet the requirements of the main test262 suite.
```

Comment:
i'd prefer to avoid hard-wrapping, if possible, but not a blocker.

Reply:
I don't mind changing this, but it should really be a lint rule. The existing text in the files is a mix of wrapping styles, and you can't see the difference when it's rendered anyway. I think we should pick a style ([here is a comprehensive blog post about pros and cons of different wrapping styles](https://mtsknn.fi/blog/4-1-wrapping-styles-for-markdown-prose-and-code-comments/)) and just automate it.

Reply:
Totally fine with doing that (i prefer a sentence per line, since "semantic linebreaks" isn't automateable)